### PR TITLE
change link in partners section

### LINKED ISSUE #374: New UX flow to match the new nav bar
Front page 'Our Hiring Partners' section goes to partners, which is now orphaned. It needs to go to 'Employers' instead.

Link wording needs to be updated to reflect this.

### PR DESCRIPTION
**Pull request by sjmog**
Closes #374

### CODE REVIEW

**Comment by nikeshashar** [comment]
@sjmog Can we say "see our partners" and then link to employers?

**Comment by pitchinvasion** [comment]
I updated the other orphan links. Anyone know a tool for checking orphan links generally?

**Comment by spike01** [comment]
Should we add a redirect for /partners -> /employers? For all our fans that bookmarked the partners page?

**Comment by pitchinvasion** [comment]
+1 @spike01 

I think maybe this PR should be expanded for the other suggestions made in website, so we're merging in a complete solution for replacing partners. I don't mind implementing the other changes if we're agreed

**Comment by sjmog** [comment]
I agree. This was meant to be a bugfix PR but if it can quickly be expanded to a complete solution let's do that

@nikesh I'm unhappy with the wording too, and how this now links to an apply section no longer targeted to students

**Comment by pitchinvasion** [comment]
Cool, @isoworg 's solution is

```
 Is there not an Apply button on the main nav?

w [11:24 AM]
Happy for one of the small buttons to be Apply (with different copy) - esp as someone mistakenly clicked on of them today. And they both point to the same link.

w [11:27 AM]
We can track how many people click Apply on the Employers page to see how influential it is on people’s decisions?
```

**Comment by sjmog** [comment]
OK, all up-to-date with those suggestions. Squirrel me at will

**Comment by pitchinvasion** [comment]
So just to be clear we want on the homepage to funnel people through to the employers page? Just if students end up on that page they have to scroll a way down to see the list of hiring partners which I imagine is why they want to be there?

I'm happy to merge this and we can open another ticket, but does it make sense to move up the list of hiring partners for this case? 

If this debate has already been had feel free to :shipit:

**Comment by sjmog** [comment]
I raised my concerns with this and Rubén said we'll watch the data to see if there are any significant drop-offs. I agree with you from a UX POV but let's go with data for the time being.

@pitchinvasion

**Comment by spike01** [comment]
I'm not totally sure what's going on, but the code looks :ship:able and let's keep an eye on the data.

File: source/partials/_apply.html.haml [hunk 1]
```diff
@@ -11,9 +11,9 @@
           %p It costs nothing, until you find the right person.
       .button-row
         %p
-          = link_to "Need something special?", "http://bit.ly/meetWB", class: "button button--horizontal"
+          = link_to "Have hiring questions?", "http://bit.ly/meetWB", class: "button button--horizontal"
```
Comment (nikeshashar) [inline source/partials/_apply.html.haml]: Should the link be changed here to `config.apply_form_url`

Comment (pitchinvasion) [inline source/partials/_apply.html.haml]: no this is a different link

**Pull request merged**